### ISSUE TITLE
poor texture for my model

### ISSUE THREAD

**Issue by miaowu99** [OpenTexture/Paint3D#24]
I followed the process in readme to run the program and wanted to texture my car model, but the result did not meet my expectations. May I ask where I ran it incorrectly？
prompt: change "monkey head" to "red car"
result:
![2024-08-13 11-39-45 的屏幕截图](https://github.com/user-attachments/assets/07cae9e1-cfce-4c23-a2b6-f77c9aaffed7)

result of stage1/init-img-0.png:
![init-img-0](https://github.com/user-attachments/assets/ce8b7cfc-9ace-494a-ba60-176a7481a15c)

axis is Zup

**Comment by miaowu99** [comment]
I found that the generated texture has a vertical flipping relationship with my original texture. Although I don't know why, after correcting this, the result is slightly normal, but still not good.
the stage1/init-img-0.png is good, but I only have front and rear views, no left and right views, right?
![init-img-0](https://github.com/user-attachments/assets/ac1dfc31-a722-4a53-aadb-90023b4cfdb7)
After stage2, the texture result looks terrible in a specific area, no matter how I rotate the mesh.
![UV_inpaint_res_0](https://github.com/user-attachments/assets/de8d5fb6-a4a5-42c6-a6fa-5e6895becbb1)
And I also found an UV_pos.png, how to use this?

**Comment by miaowu99** [comment]
before stage2:
![albedo](https://github.com/user-attachments/assets/5ae7ccff-5e3f-4b4f-b702-8f06515d1dee)
after stage2:
![UV_inpaint_res_0](https://github.com/user-attachments/assets/2d18139b-fd78-437a-a14f-0ed9153af4b3)
result:
https://github.com/user-attachments/assets/a4a424bd-5454-4a7c-96f9-6cef43407ef9

**Comment by miaowu99** [comment]
After some modifications, my results look better than before：
![2024-08-14 11-48-01 的屏幕截图](https://github.com/user-attachments/assets/f0284499-8cb1-4c36-b7b9-683bd635f36e)
1. Make sure the model is Yup and Zfront
2. Delete the original texture information in the model.
3. Set pipeline_paint3d_stage2.py line53, "export_texture_only" False, to export mesh, instead of using the original mesh to load the texture.
4. Delete paint3d_cache/* when changed the model of the same name.
Of course, there are still some issues that have not been resolved：
1. Visible textured seams.
2. Multi-faces problem.
3. Blurred and inconsistent in some areas.
Are these issues inherent to the methods themselves? Is there a plan to optimize these issues in the future, or are there any better similar methods recommend?
Thanks for the excellent work.

**Comment by Zzlongjuanfeng** [comment]
> After some modifications, my results look better than before： ![2024-08-14 11-48-01 的屏幕截图](https://private-user-images.githubusercontent.com/32644829/357671041-f0284499-8cb1-4c36-b7b9-683bd635f36e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2MDk0ODIsIm5iZiI6MTcyMzYwOTE4MiwicGF0aCI6Ii8zMjY0NDgyOS8zNTc2NzEwNDEtZjAyODQ0OTktOGNiMS00YzM2LWI3YjktNjgzYmQ2MzVmMzZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDA0MTk0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwZDQxMDBiYmQ2NDY0OGRkZThlNGY5MDcyNjUxOGNlMjAyOWE2NTMyODdkMGJiNThmMTFhNjMyY2JiN2RlZjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fzbP8EeVMfbgU9cSUa70GwRLwYNYAg6kiuhIyOKr3mI)
> 
> 1. Make sure the model is Yup and Zfront
> 2. Delete the original texture information in the model.
> 3. Set pipeline_paint3d_stage2.py line53, "export_texture_only" False, to export mesh, instead of using the original mesh to load the texture.
> 4. Delete paint3d_cache/* when changed the model of the same name.
> 
> Of course, there are still some issues that have not been resolved：
> 
> 1. Visible textured seams.
> 2. Multi-faces problem.
> 3. Blurred and inconsistent in some areas.
> 
> Are these issues inherent to the methods themselves? Is there a plan to optimize these issues in the future, or are there any better similar methods recommend? Thanks for the excellent work.

Yes, we are working on addressing the challenges of multi-face generation and inconsistencies. Thank you for your attention.

**Comment by cjkangme** [comment]
In my opinion, if there are many holes in the texture map after stage 1, most of the original texture is masked out due to the large dilate kernel size in stage 2.
So I think this could lead to a terrible-looking texture.

I think adjusting the dilate kernel size when extracting the mask could be one of the possible solutions.
